busybox/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "hub": {
+    "categories": [
+      "operating-systems"
```

Comment:
Maybe add `"developer-tools"` as well here

Reply:
We should fix ones that are incorrect in this PR, but I think extra categories should be a separate PR to prevent this one from ballooning.

Reply:
And it doesn't hurt to give image maintainers an opportunity to provide input.